src/resources/MachineLearning/UserActionHistoryConfiguration/tests/UserActionHistoryConfiguration.spec.ts

```diff
@@ -0,0 +1,41 @@
+import API from '../../../../APICore';
+import UserActionHistoryConfiguration from '../UserActionHistoryConfiguration';
+
+jest.mock('../../../../APICore');
+
+const APIMock: jest.Mock<API> = API as any;
+
+describe('UserActionHistoryConfiguration', () => {
+    let userActionHistoryConfiguration: UserActionHistoryConfiguration;
+    const api = new APIMock() as jest.Mocked<API>;
+    const serverlessApi = new APIMock() as jest.Mocked<API>;
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+        userActionHistoryConfiguration = new UserActionHistoryConfiguration(api, serverlessApi);
+    });
+
+    describe('create', () => {
+        it('should make a POST call to the User Action History Configuration base url to enable', () => {
+            userActionHistoryConfiguration.create();
+
+            expect(api.post).toHaveBeenCalledTimes(1);
```

Comment:
From what I check in the other spec files of `/src/MachineLearning/`, we also test the call arguments of the `api.post`, `api.get`, etc.

Why not here?

Reply:
There are no arguments to pass in the post :) If that's what you were wondering.